package.json

```diff
@@ -14,10 +14,11 @@
   "license": "MIT",
   "dependencies": {
     "@turf/helpers": "6.1.4",
+    "@types/xmldom": "^0.1.29",
```

Comment:
is it required or can we move this to devDependencies section?

Reply:
Yes please. xmldom it is used for parsing XML about GPX file. xmldom in dependecies and the types for TS in devDependecies. Thank you!

Reply:
yes, please move TS dependency to devDependencies section. currently it is also in dependencies section.

Reply:
Done. I updated the branch

Reply:
Thanks